src/main/java/com/spotify/folsom/client/DefaultRawMemcacheClient.java

```diff
@@ -16,6 +16,7 @@
 
 package com.spotify.folsom.client;
 
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
Agreed, but a metric seems overkill. The metric is not interesting for any other use case than testing I think.